models/tutor.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const tutorSchema = new Schema(
+  {
+    username: {type: String, required: true},
+    email: {type: String, required: true},
+    hourlyRate: {type: String, required: true},
+    programmingLanguage:{type: String, required: true}  ,
```

Comment:
Put the value for this in an array bracket, and I would move the comma in line with the end